release-packaging/Classes/FluidBufToKr.sc

```diff
@@ -0,0 +1,19 @@
+FluidKrToBuf {
+	*kr {
+		arg krStream, buffer;
+		^buffer.numFrames.do{
```

Comment:
Presumably this means that the buffer in question needs to be the expected size when the synthdef is compiled?

Reply:
This is correct. Since there's no way of knowing (I think) how many channels the krStream is, one can't check if they are equal. If the krStream is longer than buffer, it just won't all get written. I can check what will happen if the buffer size is > krStream number of channels.

Reply:
I wonder if it's worth putting in a check for the most obvious error (0-length buffer), just to reduce the foot-gun potential
```suggestion
		if(buffer.numFrames == 0) {"FluidKRToBuf: UGen will have 0 outputs!".warn}
                ^buffer.numFrames.do{
```
Or even throw? I defer to your SC wisdom...